docs/learn/digital-assets/migrate-erc20-to-lsp7.md

```diff
@@ -0,0 +1,194 @@
+---
+sidebar_label: '💰 Migrate ERC20 to LSP7'
+sidebar_position: 3
+description: Learn how to migrate your ERC20 token to the LSP7 Digital Asset standard on LUKSO.
+---
+
+# 🪙 Migrate ERC20 to LSP7
+
+[LSP7DigitalAsset](../../standards/tokens/LSP7-Digital-Asset.md) is a new token standard that offers a wider range of functionality compared to [ERC20](https://eips.ethereum.org/EIPS/eip-20), as described in the [standard section](../../standards/tokens/LSP7-Digital-Asset.md). For migrating from ERC20 to LSP7, developers need to be aware of several key differences.
+
+## Smart Contract Building
+
+Usually, to create an ERC20 token, [@openzeppelin/contracts](https://www.npmjs.com/package/@openzeppelin/contracts) package will be used, and ERC20 will be inherited.
+
+```solidity title="ERC20 Token"
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MyERC20Token is ERC20 {
+    constructor(string memory name, string memory symbol) ERC20(name, symbol) {
+        // Your constructor logic
+    }
+}
+```
+
+For LSP7, [@lukso/lsp7-contracts](https://www.npmjs.com/package/@lukso/lsp7-contracts) package will be used, and LSP7 will be inherited.
```

Comment:
Suggest adding an extra sentence with bullet points to explain briefly:

- the extra `constructor` arguments
- a link to the `constructor` method in the contract ABI.

<img width="992" alt="image" src="https://github.com/user-attachments/assets/dfe19cac-9b9c-4979-b5c2-6f7c51e2daa0">